core/domain/suggestion_services.py

```diff
@@ -550,6 +579,64 @@ def get_submitted_suggestions(user_id, suggestion_type):
     ])
 
 
+def get_user_scoring_from_model(user_scoring_model):
+    """Converts the given UserContributionScoringModel to a
+    UserContributionScoring domain object.
+
+    Args:
+        user_scoring_model: UserContributionScoringModel.
+            UserContributionScoringModelobject to be
+            converted to UserContributionScoring domain object.
+
+    Returns:
+        UserContributionScoring. The corresponding UserContributionScoring
+        domain object.
+    """
+    return user_domain.UserContributionScoring(
+        user_scoring_model.user_id, user_scoring_model.score_category,
+        user_scoring_model.score, user_scoring_model.has_email_been_sent
+    )
+
+
+def get_user_scoring(user_id, score_category):
```

Comment:
Thanks Ben, I made get_user_scoring private.

Reply:
By storage get method do you mean ``get_user_scoring_from_model`` (a) or ``UserContributionScoringModel.get`` (b)?

If (a), if a method isn't meant to be used outside of the class/service it's defined in, it should be private. It's purpose then is to support the other methods of the class/service, rather than providing a first-class behavior that callers of the class/service can use. Per your PR, it didn't seem like this function was meant to be used outside of this service. There could be multiple methods with similar behaviors where only some are part of the public API. That's okay, and in some ways ideal. Having leaner APIs provides better maintenance benefits: it makes it easier to change implementations in the future, or add additional APIs due the flexibility of having fewer API methods to consider when doing so.

If (b), the model's API is independent of this service's API since it's a different component. Whether its methods are public/private are up to the contract that it's defining in its API. This class/service can only use the public APIs available to it from the storage layer in the same way that upstream components calling into this service can only use its public API functions.